<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRUD</class>
 <widget class="QDialog" name="CRUD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>330</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Bus</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Voiture </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mini_Bus</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre des voyageurs</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mat">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Matricule </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_etat">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>400</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_Mat">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>360</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Nb">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>400</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Etat Véhicule</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_type">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type de véhicule</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
